tests/exec-tests.sh

```diff
@@ -33,8 +33,8 @@ RUN sed -i 's|</dev/tty||g' /var/lib/githooks/install.sh && \\
     # Change the base template so we can pass in the hook name and accept flags
     sed -i -E 's|echo.*Hook not run inside a git repository.*|CURRENT_GIT_DIR=".git"|' /var/lib/githooks/base-template.sh && \\
     sed -i -E 's|GITHOOKS_RUNNER=(.*)|GITHOOKS_RUNNER=\1; GITHOOKS_RUNNER="\${GITHOOKS_RUNNER:-/var/lib/githooks/base-template.sh}"|' /var/lib/githooks/base-template-wrapper.sh && \\
-    sed -i -E 's|GITHOOKS_HOOK_FOLDER=(.*)\\\|GITHOOKS_HOOK_FOLDER=\1 GITHOOKS_HOOK_FOLDER="\${HOOK_FOLDER:-\$GITHOOKS_HOOK_FOLDER}" \\\|' /var/lib/githooks/base-template-wrapper.sh && \\
-    sed -i -E 's|GITHOOKS_HOOK_NAME=(.*)\\\|GITHOOKS_HOOK_NAME=\1 GITHOOKS_HOOK_NAME="\${HOOK_NAME:-\$GITHOOKS_HOOK_NAME}" \\\|' /var/lib/githooks/base-template-wrapper.sh && \\
+    sed -i -E 's|HOOK_FOLDER=(.*)|HOOK_FOLDER="\${HOOK_FOLDER:-\1}"|' /var/lib/githooks/base-template.sh && \\
+    sed -i -E 's|HOOK_NAME=(.*)|HOOK_NAME="\${HOOK_NAME:-\1}"|' /var/lib/githooks/base-template.sh && \\
     sed -i 's|ACCEPT_CHANGES=|ACCEPT_CHANGES=\${ACCEPT_CHANGES}|'  /var/lib/githooks/base-template.sh && \\
     sed -i 's%read -r "\$VARIABLE"%eval "\$VARIABLE=\\\\\$\$(eval echo "\\\\\$VARIABLE")" # disabled for tests: read -r "\$VARIABLE"%' /var/lib/githooks/base-template.sh && \\
     sed -i -E 's|GITHOOKS_CLONE_URL="http.*"|GITHOOKS_CLONE_URL="/var/lib/githooks"|' /var/lib/githooks/cli.sh /var/lib/githooks/base-template.sh /var/lib/githooks/install.sh
```

Comment:
Maybe unrelated, but why change this url? It seems that base-template.sh and cli.sh take this url from the git config and repo remote settings, which you could just set, and install.sh already uses any value supplied through an env var or commandline option, which could just be passed, without changing the script here?

Reply:
What url do you mean `/var/lib/githooks/base-template-wrapper.sh ` to `/var/lib/githooks/base-template.sh`?
Its just to inject the proper values for like 30 tests... -> Would be good jup, to adjust these tests too maybe at the end!

Reply:
I'm talking about this line:

         sed -i -E 's|GITHOOKS_CLONE_URL="http.*"|GITHOOKS_CLONE_URL="/var/lib/githooks"|' /var/lib/githooks/cli.sh /var/lib/githooks/base-template.sh /var/lib/githooks/install.sh 

Apparently that fixates `GITHOOKS_CLONE_URL` to `/var/lib/githooks`. I'm not entirely sure *why* this is needed, but I would think you can just do e.g. `install.sh --clone-url /var/lib/githooks` when running install.sh, and do `git config --global githooks.cloneUrl /var/lib/githooks` (and maybe `git remote add origin /var/lib/githooks` after each `git init`) to just set this value normally, so you do not need this `sed` line to fixate the url.

However, this *is* really unrelated to this PR (I just noticed it here), so probably best to leave for now.

Reply:
Ah, setting global git configs is probably not ideal, but like I mentioned in https://github.com/rycus86/githooks/pull/111#discussion_r472838040, reading `githooks.cloneUrl` can probably also happen without `--global`?

Reply:
Thats true, strange I though there is something in the `exec-steps.sh`

Reply:
No this line is prefectly fine, I checked now. It makes sense. Its only replacing the **default url** incase none is set.
So it tests more paths the initial guessed in this dicussion.

Reply:
Ah, excellent point, thanks for checking!